.kitchen.vagrant.yml

```diff
@@ -39,10 +58,9 @@ platforms:
   driver_config:
     box: oracle-6.5
     box_url: https://storage.us2.oraclecloud.com/v1/istoilis-istoilis/vagrant/oel65-64.box
-- name: debian-6
+- name: oracle-7
   driver_config:
-    box: debian-6
-    box_url: https://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_debian-6.0.10_chef-provisionerless.box
+    box: boxcutter/ol72
```

Comment:
There are [bento](https://github.com/chef/bento#64-bit) boxes available for oracle, I would suggest to use them as they are maintained